libs/base/ui/action/link/src/styles/storefront.styles.ts

```diff
@@ -4,10 +4,11 @@ export const storefrontLinkStyles = css`
   :host {
     position: relative;
     display: inline-flex;
+    width: var(--oryx-link-width);
```

Comment:
Same as before, I would recommend you to describe each new css var in PR's description

Reply:
@Sinucid  I have done this in the parent PR which will be the base for release docs